BUILDING.md

```diff
@@ -12,7 +12,7 @@ Read more about setting up go in the [official docs](https://golang.org/doc/inst
 
 Install dependencies:
 ```sh
-go install github.com/spf13/cobra@latest
```

Comment:
@kfa-onewelcome `go install` doesn't work, `get -u` does the trick.

Reply:
this is completely different at my env (/bin/ accordingly), but ok, let's do this your way. You will release the new version